open-metadata-resources/open-metadata-archives/open-connector-archives/pom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <artifactId>open-metadata-archives</artifactId>
+        <groupId>org.odpi.egeria</groupId>
+        <version>1.3-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>Open Connector Archives</name>
+    <description>
+        The Open Connector Archives provide utilities for loading definitions for open connectors.
+    </description>
+
+    <artifactId>open-connector-archives</artifactId>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>repository-services-apis</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>repository-services-archive-utilities</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>open-metadata-types</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+        </dependency>
+
+        <!-- Logback used as provider for slf4j logging -->
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
```

Comment:
Is this a utility only - in which case including the binding is good, or is it used from within our servers etc? If so we should be careful with introducing an implementation - this should be done at the latest/most outside binding point possible to avoid conflicts

Reply:
Ok reviewing the code, the archive writer is simply a utility & as such it's reasonable to include logback. However  the default level shouldn't be DEBUG - which is the cause of #2321 . Additionally the 'build' is not a test step, but a packaging step, so the lifecycle binding should be packaging or similar.